src/appmixer/intelswift/core/SaveSyncTicketId/SaveSyncTicketId.js

```
@@ -0,0 +1,25 @@
+
+const { makeRequest } = require('../../commons');
+module.exports = {
+
+    async receive(context) {
+
+        const { platformType, syncId, ticketId, tenantId } = context.messages.in.content;
+
+        let body = {};
+        if(!platformType) throw 'platformType not found';
+
+        if(platformType == 'jira') body.jiraId = syncId;
```

ESLint

[error] Expected space(s) after "if".

(keyword-spacing)